modules/drivers/snowflake/test/metabase/driver/snowflake_test.clj

```diff
@@ -593,8 +593,8 @@
                              (lib/breakout dtz)
                              (lib/aggregate (lib/count)))]
           (mt/with-native-query-testing-context query
-            (is (= [["2023-09-30T00:00:00+10:00" 1]
-                    ["2023-10-01T00:00:00+10:00" 2]
-                    ["2023-11-01T00:00:00+11:00" 1]]
-                   (mt/with-temporary-setting-values [report-timezone "Australia/Sydney"]
+            (mt/with-temporary-setting-values [report-timezone "Australia/Sydney"]
```

Comment:
`mt/with-temporary-setting-values` around the `is` gives you more meaningful `testing` context when the test fails.